src/snapshot.cc

```diff
@@ -33,7 +33,7 @@ enum class SnapshotTypeTag {
   OBJECT_TAG = 0,
   IN_TABLE_TAG,
   BACK_REFERENCE_TAG,
-  PROGRAM_HEAP_REFERENCE_TAG,
+  UNUSED_TAG,
```

Comment:
I would prefer if it was removed, but since snapshots are disappearing soon anyway...